pythoncalculator/divide.py

```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
+    return x / y
```

Comment:
I think this should perhaps by `z` (or the variables in the function definition should be `x, y`)

Reply:
Thanks Neil! Corrected now.